PURL-TYPES.rst

```diff
@@ -326,6 +326,20 @@ nuget
 
       pkg:nuget/EnterpriseLibrary.Common@6.0.1304
 
+qpkg
+----
+``qpkg`` for QNX packages:
+
+- There is no default package repository: this should be implied either from
+  the ``namespace`` or using a repository base URL as ``repository_url`` 
+  qualifiers key.
+- The ``namespace`` is the vendor. It is not case sensitive and must be
+  lowercased. The default ``namespace`` is ``blackberry``.
```

Comment:
Today we don't have default namespaces for purl types and clients would need to know to substitute `blackberry` for missing namespaces. I'd suggest instead making the `namespace` required and explicitly include `blackberry` in the example below (i.e. no default namespace).